client/controllers/searchBar.js

```diff
@@ -1,42 +1,47 @@
 import locationGeoJsonPromise from '/imports/locationGeoJsonPromise';
 import { ReactiveVar } from 'meteor/reactive-var';
 
+const regexEscape = (s)=>s.replace(/[-\/\\^$*+?.()|[\]{}]/g, '\\$&')
+
```

Comment:
I wonder if it would be helpful to check this out: https://www.npmjs.com/package/escape-string-regexp